src/main/java/org/opentripplanner/middleware/bugsnag/BugsnagDispatcher.java

```diff
@@ -20,15 +20,15 @@
 import static org.opentripplanner.middleware.spark.Main.getConfigPropertyAsText;
 
 /**
- * Responsible for getting required information from Bugsnag. This is done by making calls to Bugsnag's API endpoints
+ * Responsible for getting {@link BugsnagEvent} information from Bugsnag. This is done by making calls to Bugsnag's API endpoints
  * with a valid authorization token (BUGSNAG_API_KEY).
- *
+ * <p>
```

Comment:
If using HTML inside JavaDoc, `<p>` should have a corresponding `</p>`.

Reply:
Remove HTML markup.